nntrainer/layers/activation_layer.cpp

```diff
@@ -50,17 +50,21 @@ void ActivationLayer::forwarding(sharedConstTensors in) {
   Tensor &hidden_ = net_hidden[0]->var;
   /// @note @a _act_fn is expected to work out of place and not modify @a input
   _act_fn(net_input[0]->var, hidden_);
+  if (activation_type == ActivationType::ACT_SOFTMAX)
+    tempTensor = hidden_;
```

Comment:
This is a shallow copy. tempTensor will also get updated when net_hidden[0]->var is updated.

Reply:
done.